Makefile

```diff
@@ -100,8 +100,8 @@ test: manifests generate fmt vet envtest ## Run tests.
 test-unit: envtest
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./controllers/... ./pkg/...
 
-test-e2e: binary-e2e binary-rte-e2e
-	./bin/e2e.test -ginkgo.v
+test-e2e: binary-e2e-install binary-e2e-rte binary-e2e-uninstall
+	hack/run-functests.sh
```

Comment:
let's rename this to be like `run-e2e.sh` or `run-e2e-test.sh` or something like this.

Reply:
done